src/native-automation/client/event-descriptor.ts

```diff
@@ -69,7 +70,7 @@ export default class CDPEventDescriptor {
         if (options.isNewLine)
             return '\r';
 
-        if (options.keyProperty.length === 1)
+        if (options.keyProperty.length === 1 && ![KeyModifierValues.ctrl, KeyModifierValues.alt, KeyModifierValues.meta].includes(options.modifiers))
```

Comment:
Please add the `Array.prototype.includes` method to the [native methods](https://github.com/DevExpress/testcafe-hammerhead/blob/b10bd8c688bdce0e25d8e33278bb3899dd274c2c/src/client/sandbox/native-methods.ts#L186) and use it here.

Also put this condition to the separate function:
```js
private static _getKeyDownEventText (options: SimulatedKeyInfo): any {
    if (options.isNewLine)
        return '\r';

    if (this._keyModifier(options)
        return options.keyProperty;
    ...
}

```